uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1787,13 +1787,13 @@ sub CreateMRICandidates {
     chomp($User);
     $candID = NeuroDB::MRI::createNewCandID($dbhr) unless $candID;
     my %record = (
-        CandID               => $subjectIDsref->{'CandID'},
-        PSCID                => $subjectIDsref->{'PSCID'},
-        DoB                  => $subjectIDsref->{'PatientDoB'},
-        ProjectID            => $subjectIDsref->{'ProjectID'},
-        Sex                  => $sex,
-        RegistrationCenterID => $centerID,
-        UserID               => $User,
+        CandID                => $subjectIDsref->{'CandID'},
+        PSCID                 => $subjectIDsref->{'PSCID'},
+        DoB                   => $subjectIDsref->{'PatientDoB'},
+        Sex                   => $sex,
+        RegistrationCenterID  => $centerID,
+        RegistrationProjectID => $subjectIDsref->{'ProjectID'},
```

Comment:
Given that the default ProjectID would be undef, I wonder whether the if statement line 1758 should not be modified by `!defined $subjectIDsref->{'ProjectID'} and $pscid eq 'scanner'` since no ProjectID should be assigned to scanner candidates.

@laemtl thoughts?

Reply:
I believe the change [here](https://github.com/aces/Loris/blob/973fb261e420ae23696ddda628bfe2831260353e/SQL/Archive/24.0/2019-07-01_fix_project_in_session.sql) will not allow scanner to have no ProjectID.